daemon/api.go

```diff
@@ -797,6 +798,17 @@ func (inst *snapInstruction) modeFlags() (snapstate.Flags, error) {
 	return modeFlags(inst.DevMode, inst.JailMode, inst.Classic)
 }
 
+func (inst *snapInstruction) installFlags() (snapstate.Flags, error) {
```

Comment:
Do we need a separate method for this?  Feels like it might be in modeFlags itself.